test/api/controllers/realtime.test.js

```diff
@@ -116,15 +116,13 @@ describe('RealtimeController', () => {
     it('should throw an error if body is not provided',() => {
       request.input.body = null;
 
-      should(() => {
-        realtimeController.unsubscribe(request);
-      }).throw(BadRequestError, { id: 'api.assert.body_required' });
+      should(realtimeController.unsubscribe(request))
```

Comment:
must be either returned or awaited, otherwise test will always pass even if the condition isn't verified